tools/pkcommon.py

```diff
@@ -0,0 +1,55 @@
+# vim: ts=4 et sw=4 sts=4 :
+import sys
+from pathlib import Path
+
+PROFILES = ("easy", "standard", "restrictive")
+profile_dir = Path(__file__).parent.with_name("profiles")
+
+def printerr(*args, **kwargs):
+    kwargs["file"] = sys.stderr
+    print(*args, **kwargs)
+
+
+def getProfilePath(which):
+    base = "polkit-default-privs.{}".format(which)
+    return profile_dir / base
+
+
+class ProfileEntry:
+
+    path = ""
+    line = ""
+    linenr = 0
+    action = ""
+    settings = tuple()
+
+
+def parseProfile(path, callback):
```

Comment:
I feel the code would be cleaner if you'd `yield` the individial entries instead of calling a callback function on them.